### DISCUSSION TITLE
"For You" beta feed only enumerating bot activity

### DISCUSSION THREAD

**Discussion by shollyman** [community/community#14736] [Other Feature Feedback, Questions, & Ideas]
I don't know what it says about me, but the beta "for you" feed has decided that `renovate-bot` (which is configured for managing dependencies on repos I work with) is a frequent collaborator, and shows me nothing except its contributions to various repositories.

It would be useful if bot activity was excluded from this feed.  As it stands, this feed is less valuable than the previous feed which showed me what actual human collaborators have been doing.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
